content/en/docs/contribute/start.md

```diff
@@ -196,7 +196,7 @@ to base your work on. Use these guidelines to make the decision:
 
 - Use `master` for fixing problems in content that is already published, or
   making improvements to content that already exists.
-  - Use a release branch (such as `dev-{{< release-branch >}}` for the {{< release-branch >}} release) to document upcoming features
+  - Use a release branch (such as `{{< dev-branch >}}` for the upcoming {{< future-version >}} release) to document upcoming features
   or changes for an upcoming release that is not yet published.
```

Comment:
nit: possible edit:
Use a release branch (such as `{{< dev-branch >}}` for the {{< future-version >}} release) to document new features or changes assigned to the upcoming release.